src/streamlink/plugins/twitch.py

```diff
@@ -399,9 +387,21 @@ class Twitch(Plugin):
         validate.all({}, _schema_metadata_empty)
     ))
 
+    QUALITY_WEIGHTS = {
+        "source": 1080,
+        "1080": 1080,
+        "high": 720,
+        "720": 720,
+        "medium": 480,
+        "480": 480,
+        "360": 360,
+        "low": 240,
+        "mobile": 120,
```

Comment:
Pretty sure the lowest (outside of `audio_only`) is now 160p, there's also no longer a `low` option.

Reply:
I don't think this quality mapping is relevant anymore. It just maps the old quality names to the new ones by making a guess, nothing more. This was added during a time, where both were still available on Twitch. I had the same issue in the Twitch GUI and it still maps the qualities to the old names there, so that user customizations work, as the "new" quality names are dynamic. For Streamlink, this isn't an issue. I just didn't want to change anything that doesn't need change. If it needs to be removed, this can be done later. Or I'll see later if it's really needed and then remove it here.